### PR TITLE
Convert datasets to use kagglesdk

### PR DESCRIPTION
**Pull request by stevemessick**
I have some `TODO`s that won't be done until Swagger is removed.

Ignore everything in `kagglesdk`.

The unit tests were not reformatted earlier.

```bash
$ yapf --version
yapf 0.40.2
```
The version is the same as specified in #634, so I don't know why some indentation changed.

### CODE REVIEW

**Comment by calderjo** [comment]
any chance you can break this down into a smaller prs (maybe two). it's a bit difficult to see, what actually changed and what is just reformatted.

maybe one PR can be just reformat, then the other is the change you introduced.

**Comment by stevemessick** [comment]
Thanks for taking a look, @calderjo 
> any chance you can break this down into a smaller prs

I tried, but it didn't work, and now I've got two branches that seem to have the same changes. Once I get this one merged then I'll rebase the new one and see what's in it.

The review can be simplified, though. You can ignore everything in `kaggle` and `kagglesdk` since those directories were generated. In `src`, there are two files and they are fairly easy to read. You can ignore `unit_tests.py` because all I did was add some ordering and additional asserts. That's the one that got new indentation and is hard to compare.

File: src/kaggle/api/kaggle_api_extended.py [hunk 1]
```diff
@@ -1740,41 +1751,67 @@ def dataset_create_version(self,
...
-        files=[],
-        convert_to_csv=convert_to_csv,
-        category_ids=keywords,
-        delete_old_versions=delete_old_versions)
+    body = ApiCreateDatasetVersionRequestBody()
+    body.version_notes=version_notes
+    body.subtitle=subtitle
+    body.description=description
+    body.files=[]
+    # body.convert_to_csv=convert_to_csv
```
Comment (calderjo) [inline src/kaggle/api/kaggle_api_extended.py:1759]: is this comment needed?

File: src/kaggle/api/kaggle_api_extended.py [hunk 2]
```diff
@@ -1740,41 +1751,67 @@ def dataset_create_version(self,
...
-                self.datasets_create_version_with_http_info(
-                    owner_slug, dataset_slug, request)))
+          self.process_response(
+            self.datasets_create_version_with_http_info(
+              owner_slug, dataset_slug, request)))
 
       return result
 
+  def _api_dataset_new_file(self, file):
+    # TODO Eliminate the need for this conversion
```
Comment (calderjo) [inline src/kaggle/api/kaggle_api_extended.py:1810]: is there a bug listing the blocker for this

File: src/kaggle/api/kaggle_api_extended.py (hunk 1 above)
Comment (stevemessick) [reply src/kaggle/api/kaggle_api_extended.py:1759]: No, good catch.

File: src/kaggle/api/kaggle_api_extended.py (hunk 2 above)
Comment (stevemessick) [reply src/kaggle/api/kaggle_api_extended.py:1810]: Not as such. The blocker is removing all Swagger code. That is, it is a note-to-self that I need to finish up a few things before calling it done.

File: src/kaggle/models/kaggle_models_extended.py [hunk 3]
```diff
@@ -115,9 +115,14 @@ def __repr__(self):
 class File(object):
 
   def __init__(self, init_dict):
-    parsed_dict = {k: parse(v) for k, v in init_dict.items()}
-    self.__dict__.update(parsed_dict)
-    self.size = File.get_size(self.totalBytes)
+    try: # TODO Remove try-block
```
Comment (calderjo) [inline src/kaggle/models/kaggle_models_extended.py:118]: what's the issue with try catch, add context

File: src/kaggle/models/kaggle_models_extended.py [hunk 4]
```diff
@@ -165,13 +170,18 @@ def __repr__(self):
 class ListFilesResult(object):
 
   def __init__(self, init_dict):
-    self.error_message = init_dict['errorMessage']
-    files = init_dict['datasetFiles']
+    try: # TODO Remove try-block
```
Comment (calderjo) [inline src/kaggle/models/kaggle_models_extended.py:173]: ditto.

File: src/kaggle/models/kaggle_models_extended.py (hunk 3 above)
Comment (stevemessick) [reply src/kaggle/models/kaggle_models_extended.py:118]: Again, this is just another note-to-self to address after Swagger is removed.

**Comment by calderjo** [comment]
thanks, was not aware that kagglesdk is auto generated.

**Review by calderjo — APPROVED** [review]
LGTM, just a few question about comments

**Pull request merged**